README.md

```diff
@@ -20,8 +20,31 @@ Scientific code repositories contain valuable metadata which can be used to enri
 
 To install the dev version from github:
 
+For usage:
+
+```shell
+pip install gimie
```

Comment:
This installs the stable version from PyPI. The dev version would be:
`pip install git+https://github.com/SDSC-ORD/gimie.git@main#egg=gimie`

Reply:
I added now both options: to install the stable version or the dev version: see below.